variables.tf

```diff
@@ -56,7 +56,7 @@ variable "region" {
   }
 }
 variable "kms_endpoint_url" {
-  description = "The URL of the KMS endpoint to use when configuring KMS encryption. The Hyper Protect Crypto Services endpoint URL format is `https://api.private.<REGION>.hs-crypto.cloud.ibm.com:<port>` and the Key Protect endpoint URL format is `https://<REGION>.kms.cloud.ibm.com`."
+  description = "The URL of the KMS endpoint to use when configuring KMS encryption. The Hyper Protect Crypto Services endpoint URL format can be found at https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-regions#new-service-endpoints, and the Key Protect endpoint URL format is `https://<REGION>.kms.cloud.ibm.com`."
```

Comment:
Can you also find a key protect doc for the same reason of not having to maintain that. Also the format shown displayed currently is the global public endpoint. People may wish to use regional, and / or private so better to just link to a doc.